packages/plugin-ext/src/plugin/command-registry.ts

```diff
@@ -181,7 +182,7 @@ export class CommandsConverter {
             this.isSafeCommandRegistered = true;
         }
 
-        if (command.command && command.arguments && command.arguments.length > 0) {
+        if (command.arguments && command.arguments.length > 0) {
```

Comment:
`if (command.arguments?.length) {` ?

Reply:
You and your newfangled operators ;-) Will do.

Reply:
Actually, I'd rather not: I can't write `command.arguments?.length > 0)` and I dislike using "falsy" for zero. Also: minimal change.

Reply:
As you wish 👍